src/MainPage/Questions/FAQuestion/index.jsx

```diff
@@ -33,7 +33,7 @@ class FAQuestion extends Component {
         <div
           className="answer"
           style={{
-            height: this.state.showAnswer ? "100%" : 0
+            "max-height": this.state.showAnswer ? "500px" : 0
```

Comment:
Is there a reason you chose to limit the height to 500px? The answer to "Can I be a mentor, judge, or volunteer?" is cut off.

Reply:
I thought 500 was adequate, my bad will boost to 600px

Height transitions don't work with ``auto`` or ``100%``, so an alternative is to instead transition max-height between 0 and a set value, so the div will reach "auto" size, and still retain proper animation.

Reply:
Fixed the issue